### ISSUE TITLE
BUG - [Sev-3] - [Android] - Talkback doesn't read in logical order on Travel Claim details (Travel reimbursement claim section)

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va-mobile-app#13240]
## What happened?

See screen shots below showing how Talkback skips around on Travel Claims details page in the Travel reimbursement claim section.

## Specs:

- Device: Android
- OS: Android
- User Account:  Nolle Barakat
- Accessibility State: Talkback on.

## Steps to Reproduce

1. Log in and navigate to Health tab.
2. Go to an appointment with a green  "Confirmed" status bubble.
3. Scroll down on the appointment details screen to the Travel reimbursement claim section and read that section with Talkback on.

## Desired behavior

Talkback should have read in a logical top-down order without skipping sections.
## Acceptance Criteria

Talkback reads in a logical top-down order without skipping sections.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact:  Low 
- Frequency:  Low 

- 3 - Low

## Linked to Story

Found while testing this PR:  https://github.com/department-of-veterans-affairs/va-mobile-app/pull/13158

## Screen shot(s) and additional information

<img width="1497" height="871" alt="Image" src="https://github.com/user-attachments/assets/1823f684-474c-4766-b915-5c9cc2fe0d10" />

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [x] Steps to reproduce are defined
- [x] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))
